net/lux/Portfile

```diff
@@ -3,225 +3,24 @@
 PortSystem          1.0
 PortGroup           golang 1.0
 
-go.setup            github.com/iawia002/lux 0.18.0 v
+go.setup            github.com/iawia002/lux 0.24.1 v
+go.offline_build    no
+revision            0
+
 categories          net
-maintainers         {l2dy @l2dy} openmaintainer
+installs_libs       no
 license             MIT
+maintainers         {l2dy @l2dy} \
+                    {gmail.com:herby.gillot @herbygillot} \
+                    openmaintainer
 
 description         fast and simple video downloader
 
 long_description    Lux is a ${description} built with Go.
 
-checksums           ${distname}${extract.suffix} \
-                        rmd160  ef8fe3e8e8b4df2170f318bbd39ad504bece6679 \
-                        sha256  b3e70a63ad5053b37f95f717c97df6d8d57e4b63abf46df88a06d1cca4fa5443 \
-                        size    103624
-
-go.vendors          gopkg.in/yaml.v3 \
```

Comment:
Why was this removed? Is vendoring dependencies no longer possible?

Reply:
Ran into some issues trying to vendor this version, and just chose not to deal with the struggle of getting it to work.